src/patternfly/components/Tabs/tabs-scroll-button.hbs

```diff
@@ -1,7 +1,10 @@
-<button class="pf-c-tabs__scroll-button{{#if tabs-scroll-button--modifier}} {{tabs-scroll-button--modifier}}{{/if}}"
+<button class="pf-c-tabs__scroll-button{{#if tabs-scroll-button--IsDisabled}} pf-m-disabled{{/if}}{{#if tabs-scroll-button--modifier}} {{tabs-scroll-button--modifier}}{{/if}}"
```

Comment:
do you need `{{#if tabs-scroll-button--IsDisabled}} pf-m-disabled{{/if}}`? We would only use that if this weren't a `button` and the `disabled` attribute weren't available. The styling can come from `:disabled`